DigitalLearningSolutions.Data.Tests/DigitalLearningSolutions.Data.Tests.csproj

```diff
@@ -9,13 +9,17 @@
 
   <ItemGroup>
     <None Remove="TestData\ActivityDataDownloadTest.xlsx" />
+    <None Remove="TestData\CourseDelegateExportCurrentDataDownloadTest.xlsx" />
     <None Remove="TestData\DelegateUploadTest.xlsx" />
   </ItemGroup>
 
   <ItemGroup>
     <Content Include="TestData\ActivityDataDownloadTest.xlsx">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
+    <Content Include="TestData\CourseDelegateExportCurrentDataDownloadTest.xlsx">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Content>

```

Comment:
Does this happen automatically when you add the test data file?

Reply:
Yeah, I'm not sure what the Remove= one at the top does, but we definitely need the lower changes so that the files are copied to the test build directory so that the can be used when the tests runs.